ClientApp/src/types/sprint.ts

```diff
@@ -3,8 +3,12 @@ import { TaskData } from './task';
 export type SprintData = {
   id: number;
   name: string;
-  dueDate: Date;
-  complete: boolean;
+  startDate: Date;
+  endDate: Date;
+  pointsCompleted: number;
+  pointsAttempted: number;
+  projectId: number;
+  isBacklog: boolean;
```

Comment:
After this PR, we'll probably want a isHidable attribute as well that we'd keep track of.